alembic/autogenerate/rewriter.py

```diff
@@ -23,41 +23,40 @@
     from ..runtime.environment import _GetRevArg
     from ..runtime.migration import MigrationContext
 
+    ProcessRevisionDirectiveFn = Callable[
```

Comment:
**Federico Caselli** (CaselIT) wrote:

It probably makes sense to move the type aliases outside of the type checking if

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4990

Reply:
I got a `NameError: name 'MigrationContext' is not defined` when doing so.

Reply:
You can make that a string, similar to how migration script is

Reply:
Ah, I see. Thank you!